appveyor.yml

```diff
@@ -34,12 +31,15 @@ install:
   - conda update -q conda
   - conda config --add channels conda-forge
   - conda info -a
-  - "conda create -q -n test-environment python=%PYTHON_VERSION%"
-  - activate test-environment
-  - conda install pandas shapely=1.5 fiona pyproj rtree matplotlib descartes
-  - conda install pytest mock
+  - "conda create --quiet --name test-environment python=%PYTHON_VERSION% pandas msinttypes --file requirements.txt --file requirements.test.txt"
 
 test_script:
   - activate test-environment
   - conda list
+  - set GEOS_LIB=%CONDA_PREFIX%\\Library\\lib
+  - set GEOS_HEADERS=%CONDA_PREFIX%\\Library\\include
+  # When building Python 2.7 we need msinttypes to provice stdint.h,
+  # but that still lacks stdbool.h
+  - copy stdbool.h %CONDA_PREFIX%\\Library\\include
```

Comment:
We don't need to leave this file in the root directory, but we need to copy it over to provide `stdbool.h` for vc 9, same story as `sdtint.h`.

Note that the compilation still fails with `vc 9`, maybe it is an easy fix, maybe you have to abandon Python 2.7 on Windows. I did in a few projects of mine.